dashboard/lib/controllers/api/v1/pd/workshop_score_summarizer.rb

```diff
@@ -81,6 +96,7 @@ def get_score_for_workshops(workshops, facilitator_breakdown = false)
         survey_response.symbolize_keys.each do |k, v|
           if OVERALL_SUCCESS_QUESTIONS.include?(k)
             score = PdWorkshopSurvey::AGREE_SCALE_OPTIONS.index(v) + 1
+          elsif FREE_RESPONSE_QUESTIONS.include?(k)
           else
```

Comment:
Is this if clause intentionally empty? (betw lines 99-100 on the new file)
If so, why not chance the else to `elsif !FREE_RESPONSE_QUESTIONS.include?(k)` instead? Or at least add a comment here explaining that these are being intentionally skipped.